### DISCUSSION TITLE
a very amazing enhance picture algorithm by google

### DISCUSSION THREAD

**Discussion by mizka76** [tannerhelland/PhotoDemon#366] [General]
hello! 
 today I read  this article about enhancing photos .... 
https://ai.googleblog.com/2021/07/high-fidelity-image-generation-using.html?m=1

I thought it is very interesting and an amazing result !

**Comment by tannerhelland** [comment]
Very amazing indeed!  🤯

**Reply by marvinkim1** [reply]
Astonishing! Details and professional. I've tried [photo enhancement](https://www.fotor.com/features/one-tap-enhance) here. Are they the same procedure processing?
